controller/array_action/array_mediator_xiv.py

```diff
@@ -126,15 +126,13 @@ def _get_source_object_wwn(self, cli_object):
         return cli_object.copy_master_wwn
 
     def _generate_volume_response(self, cli_volume):
-        # vol_copy_type and copy_master_wwn were added in a9k. In xiv they didn't exist
-        is_copy = hasattr(cli_volume, "vol_copy_type") and cli_volume.vol_copy_type == "Copy"
-        copy_src_object_wwn = cli_volume.copy_master_wwn if is_copy else None
+        source_object_wwn = cli_volume.copy_master_wwn if not self._is_gen3(cli_volume) else None
```

Comment:
What's your opinion regarding double negative?

My suggestion:
```suggestion
        source_object_wwn = None if self._is_gen3(cli_volume) else cli_volume.copy_master_wwn
```

Reply:
I don't see a **double** negative here, and I think gen3+None should remain on the right, because they are (literally) marginal

Reply:
not + else equals double.

@ArbelNathan WDYT?

Reply:
I agree with ori

Reply:
As you wish